effects.json

```diff
@@ -1,99 +1,94 @@
 {
-  "BLOCK_CRAFTER_CRAFT": {
+  "SOUND_CRAFTER_CRAFT": {
     "name": "CRAFTER_CRAFT",
     "type": "soundEvent"
   },
-  "BLOCK_CRAFTER_FAIL": {
+  "SOUND_CRAFTER_FAIL": {
     "name": "CRAFTER_FAILED",
     "type": "soundEvent"
   },
-  "BLOCK_DISPENSER_DISPENSE": {
+  "SOUND_DISPENSER_DISPENSE": {
     "data": 1000,
     "name": "SOUND_CLICK",
     "type": "soundLevel"
   },
-  "BLOCK_DISPENSER_FAIL": {
+  "SOUND_DISPENSER_FAIL": {
     "data": 1200,
     "name": "SOUND_CLICK_FAIL",
     "type": "soundLevel"
   },
-  "BLOCK_DISPENSER_LAUNCH": {
+  "SOUND_DISPENSER_PROJECTILE_LAUNCH": {
     "data": 1000,
     "name": "SOUND_CLICK",
     "type": "soundLevel"
   },
-  "ENTITY_ENDEREYE_LAUNCH": {
-    "identifier": "minecraft:player",
-    "name": "THROW",
-    "type": "soundEvent"
-  },
```

Comment:
why was this removed?

Reply:
It's no longer present in the game. I can re-add it, however it would never be used.

Reply:
Ah, I see. Makes sense to remove it; thanks for the clarification!